docs/UserGuide.adoc

```diff
@@ -165,16 +165,24 @@ Displays the list of spending tagged with `groceries` in `descending order by da
 
 === Find by keyword : `find`
```

Comment:
Update 'Find by keyword' to something similar to 'Find by name, cost range, date range, remark or tags'.